### ISSUE TITLE
Warning after Vite 5 update

### ISSUE THREAD

**Issue by EricRovell** [sveltetools/svelte-pathfinder#43]
After update to Vite v5.0 there are warning in the console:

```
[vite-plugin-svelte] WARNING: The following packages have a svelte field in their package.json but no exports condition for svelte.

svelte-pathfinder@4.7.1

Please see https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/faq.md#missing-exports-condition for details.
```

Everything builds and works just fine.

**Comment by PaulMaly** [comment]
Actually, now I have no idea why this could happens. Will need to investigate it before the answer or you could provide some solution in PR.

**Comment by PaulMaly** [comment]
@EricRovell possibly fixed after #44 merge in 4.8.1 version. Please take a look. Thanks to @mishankov

**Comment by Greenheart** [comment]
Confirmed fix. Thanks for the update! :)
